src/documentation/components/ComponentTabExample.vue

```diff
@@ -0,0 +1,139 @@
+<template>
```

Comment:
this component has a lot of code from `ComponentExample`, could we reuse that one? And e.g. add a new property "tab" that if true the new code implemented here would be implemented on there.

Reply:
We actually thought of creating a seperate file for this approach as we will end up having more conditionals on existing ComponentExample if we use it. Tried to keep it seperate and clean so that we can made additional chnages in future.